README.md

```diff
@@ -47,22 +46,34 @@ also use URL syntax to provide further parameters and any other options
 Will consume the incoming message object that contains `body` with the payload.
 If the exchange doesn't exist it will be created on start.
 
-Optionally you can use `#` or `*` to wildcard. For more information check the
-tutorial provided at the [RabbitMQ site](http://www.rabbitmq.com/tutorials/tutorial-five-javascript.html).
+#### Configuration Fields
+* **Exchange** - (string, required): Exchange name where you want to get messages
+* **Binding Keys**  - (string, optional): Optionally you can use `#` or `*` to wildcard. For more information check the tutorial provided at the [RabbitMQ site](http://www.rabbitmq.com/tutorials/tutorial-five-javascript.html).
+* **Don't decrypt payload**  - (string, optional): If checked payload will be not decrypted
+* **Reconnect Timeout** - (string, optional, 5 by default): In case of errors how long to wait until retry is seconds
+* **Reconnect Attempts** - (string, optional, 12 by default): How many times try to reconnect before throw error
+
 
 ## Actions
 
 ### Publish
-
 Will publish the messages into an exchange. This exchange will be created on
 start if it doesn't exists.
 
+#### Configuration Fields
+* **Exchange** - (string, required): Exchange name where you want to send message to
+* **Don't encrypt payload** - (checkbox, optional): If checked payload will be not encrypted
+* **Content-Type** - (string, optional): Content-Type of pushed payload, default is `application/octet-stream`
+* **Reconnect Timeout** - (string, optional, 5 by default): In case of errors how long to wait until retry is seconds
+* **Reconnect Attempts** - (string, optional, 12 by default): How many times try to reconnect before throw error. 12 by default
+
+
 ## Known limitations
 
 Following limitations of the component are known:
 *   You can not publish to the default exchange.
 *   All published exchanges are `topic` exchanges by default. However, with the `topic` exchanges one can emulate `direct` and `fanout` exchanges.
-
+*   Real-time flow is preferred for trigger to avoid data loss
```

Comment:
```suggestion
*   Real-time flow is recommended for trigger to avoid data loss
```

Reply:
I'd move this directly to the trigger section. So that the users have less chances to miss this important note